doc/classes/ArrayMesh.xml

```diff
@@ -108,7 +108,7 @@
 		<method name="regen_normal_maps">
 			<return type="void" />
 			<description>
-				Will regenerate normal maps for the [ArrayMesh].
+				Will regenerate the tangents for each of the [ArrayMesh]'s surfaces'.
```

Comment:
```suggestion
				Regenerates tangents for each of the [ArrayMesh]'s surfaces.
```

"Will verb" is not a usual form for our class reference, so it's better to change it. (Same is true for the description above, if you would like to tackle it here).

Also a typo at the end.

Reply:
> "Will verb" is not a usual form for our class reference, so it's better to change it. (Same is true for the description above, if you would like to tackle it here).
> 
> Also a typo at the end.

Thank you for the guidance. I fixed the description of the other method as you suggested. Would it be okay if I added the doc references to ArrayMesh::regen_normal_maps to the places mentioned in #52444 in this PR? Or should I create a seperate PR?

Reply:
@matt08-prog As far as I can tell, the discussion suggests to reference this method in some tutorials. Tutorials are not a part of this repository, but of the [godot-docs](https://github.com/godotengine/godot-docs) repository instead. So you can't make the necessary changes in this PR, you need to open a new one in the docs repo for those.